### DISCUSSION TITLE
Relase Notes on Hebrew on App connect ios

### DISCUSSION THREAD

**Discussion by sailes-coding** [codemagic-ci-cd/codemagic-docs#2165] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

i create the release notes like 
[
  {
    "language": "he",
    "text": "minor bug fixes",
    "whatsNew": "minor bug fixes"
  }
]

but its not working ios app store publish 

### Expected results

auto fill whats new in app store

### Actual results

not filling

### Build id (optional)

654211d3f3ec1c56e665db2f

**Comment by himesh-cm** [comment, marked as answer]
Hi @sailes-coding , 
Please find here the supported languages [here](https://developer.apple.com/documentation/appstoreconnectapi/betabuildlocalizationcreaterequest/data/attributes), Hebrew is not supported yet.
